From c1f9dc7826f2bfb6d4008c5bf0fb4cffe4689658 Mon Sep 17 00:00:00 2001
From: Marek Belisko <marek.belisko@streamunlimited.com>
Date: Tue, 15 Apr 2014 09:37:32 +0200
Subject: [PATCH] s800:board: Remove s_init()

With latest u-boot we get error about multiple defined functions

Signed-off-by: Marek Belisko <marek.belisko@streamunlimited.com>
---
 board/streamunlimited/stream800/board.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/board/streamunlimited/stream800/board.c b/board/streamunlimited/stream800/board.c
index c31cbab..8bfaaf3 100755
--- a/board/streamunlimited/stream800/board.c
+++ b/board/streamunlimited/stream800/board.c
@@ -98,14 +98,6 @@ static void board_printf_info(void)
 		   stream800CarrierBoardTypeStrings[board.carrierBoardType]);
 }
 
-/* workaround: this is called from 'arch/arm/cpu/armv7/lowlevel_init.S'
- * even if is CONFIG_SKIP_LOWLEVEL_INIT not defined (see 'include/configs/stream800.h')
- * So can be empty (all low level init is done by the SPL).
- */
-void s_init(void)
-{
-}
-
 static void board_detect(struct Stream800Board *board);
 
 void set_board_interface(struct StreamBoardInterface * interface) {
-- 
2.7.4

